lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1930,41 +1975,80 @@ void SwiftASTContext::AddExtraClangCC1Args(
     return;
   }
 
-  // Clear module cache key and other CAS options to load modules from disk
-  // directly.
-  invocation.getFrontendOpts().ModuleCacheKeys.clear();
-  invocation.getCASOpts() = clang::CASOptions();
-
-  // Ignore CAS info inside modules when loading.
-  invocation.getFrontendOpts().ModuleLoadIgnoreCAS = true;
-
   // Add options to allow clang importer to do implicit module build.
```

Comment:
// Add options to allow ClangImporter to do implicit module build.

makes it easier to search